packages/story-editor/src/components/dropTargets/provider.js

```diff
@@ -176,39 +175,29 @@ function DropTargetsProvider({ children }) {
           resource,
         });
       }
-      const finalizeDrop = (baseColor = null) => {
-        if (baseColor) {
-          combineArgs.firstElement.resource.baseColor = baseColor;
```

Comment:
Is `baseColor` still properly added to `firstElement`?

Reply:
baseColor is added to all resources...

Reply:
That's not what I meant.

Setting `combineArgs.firstElement.resource.baseColor` still seems needed.

Reply:
Is it needed? We now generate base color for all elements and not just the first element.

Reply:
This is not about color generation.

This is about copying the baseColor from one element to the other